tests/core/test_grad.py

```diff
@@ -27,9 +27,15 @@ def _test_audio_grad(attr: str, target=True, kwargs: dict = {}):
                 # If necessary, propagate spectrogram changes to waveform
                 if result.stft_data is not None:
                     result.istft()
-                result.audio_data.sum().backward()
+                if result.audio_data.dtype.is_complex:
+                    result.real().sum().backward()
+                else:
+                    result.audio_data.sum().backward()
             else:
-                result.sum().backward()
+                if result.dtype.is_complex:
+                    result.real.sum().backward()
```

Comment:
Should this be `real()` (as opposed to `real`) like above?

Reply:
Lol guess I read that wrong:
<img width="482" alt="Screen Shot 2024-02-01 at 10 41 14 AM" src="https://github.com/descriptinc/audiotools/assets/4022646/3bfcfffc-872f-4617-a2d1-df0ac1e103e7">

Reply:
Shouldn't this be updated in line 32 too?

Reply:
Oh huh, curious that Python didn't complain. Will change and see if the build breaks